plugins/imagetools.md

```diff
@@ -86,6 +86,27 @@ tinymce.init({
 });
 ```
 
+### `imagetools_fetch_image`
+
+This option can be used to define a custom fetch function, which provides another way to access images in complex situations. The function will be passed the HTML element of the image to be fetched and should return a `Blob` containing the content.
+
+**Type:** `Function`
+
+##### Example
+
+```js
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  toolbar: "image",
+  plugins: "image imagetools",
+  imagetools_fetch_image: function (img) {
```

Comment:
It should return a promise you can change this to say return Promise.resolve(new Blob(...)) maybe.

Reply:
I opted against `Promise.resolve` as it's unlikely someone would actually end up using that and instead used the more async method. So that should hopefully be a more realistic example.